de.j0chn.nextcloud_password_client.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should maintain it at https://gitlab.com/j0chn/nextcloud_password_client/. (Also the icon file.)

Reply:
the icon file is already at https://gitlab.com/j0chn/nextcloud_password_client/-/blob/main/assets/npc_logo.png

Reply:
Okay, so in the end I will only have flathub.json and com.gitlab.j0chn.nextcloud_password_client.json in my github repo for flathub, right?

Reply:
basically. And maybe some patches as needed, but as upstream, patches should be temporary.